docs/api/upgrade-assistant/reindexing.asciidoc

```diff
@@ -6,7 +6,16 @@
 
 experimental[] Start a new reindex or resume a paused reindex.
 
-Start a new reindex or resume a paused reindex.
+Start a new reindex or resume a paused reindex. Following steps are performed during 
+a reindex task:
+
+. Setting the index to read-only
+. Creating a new index
+. {ref}/docs-reindex.html[Reindexing] documents into the new index
+. Creating an index alias for the new index
+. Deleting the old index
```

Comment:
This list doesn't contain 2 steps: stopping and starting "index group services, only applies to some indices" which happens for ml and watcher indices. This code will be removed from UA this release because system indices will have their own separate deprecation resolution process.